src/main/java/org/jenkins/tools/test/hook/MinaSshdApi.java

```diff
@@ -16,11 +16,6 @@ protected String getParentProjectName() {
         return "mina-sshd-api-parent";
     }
 
-    @Override
-    protected String getPluginFolderName(UpdateSite.Plugin currentPlugin){
-        return currentPlugin.getDisplayName();
-    }
```

Comment:
Small bug fix: this should never have been overridden in the first place (my bad!) but happened to work as long as we were using the short name as the display name. Now that we fixed the other bug, we have to fix this bug as well.